packages/deployer/src/compiler.ts

```diff
@@ -162,49 +171,69 @@ export class Compiler {
         }
         const solcInstance = solc.setupMethods(requireFromString(solcjs, compilerBinFilename));
 
-        logUtils.log(`Compiling ${fileName} with Solidity v${solcVersion}...`);
-        const source = this._contractSources[fileName];
-        const input = {
-            [fileName]: source,
-        };
-        const sourcesToCompile = {
-            sources: input,
+        logUtils.log(`Compiling ${contractName} with Solidity v${solcVersion}...`);
+        const source = this._contractSources[contractName].source;
+        const absoluteFilePath = this._contractSources[contractName].absoluteFilePath;
+        const standardInput: solc.StandardInput = {
+            language: 'Solidity',
```

Comment:
Should we make an enum for this?

Reply:
Nah. Let's do that when we have more than one value.